css/utilities/padding.css

```diff
@@ -1,3 +1,7 @@
+.padding-1-2-3 {
+  padding: 1rem 2rem 3rem;
+}
+
```

Comment:
Haha. I also sometimes have to urge to create a very specific utility class. 

Also, tell me which one of the following looks okay?

<img src="https://user-images.githubusercontent.com/38975782/152491420-434ab67f-8937-42d1-8eea-029c69730caa.png" width="500" height="auto">

<img src="https://user-images.githubusercontent.com/38975782/152491502-496caa5c-d299-4056-8484-0a2e7313de46.png" width="500" height="auto">

Reply:
also, play with `object-position` for the image.

Reply:
I think if we will use the first one, then the user's first attention will go on Image rather than Log In Button. As we see face first, that's why I used second and not showed complete face.

And thanks for reminding me about this property, I will play.

Reply:
oh, no. the comparison was for padding. not for the image.

Reply:
ohh!😅I think the first one. more padding above `Log In` and less below `Sign Up`. I will change, thanks.

Reply:
Yeah.:) so my point was, if we try to keep the padding around the element consistent, then we can avoid giving very specific utility classes.